### ISSUE TITLE
PTA Formatting

### ISSUE THREAD

**Issue by antonyjsmith** [maxwelllucas/ML-HLX#5]
From @samanthapope 

I’ve received some test PTA’s this morning for F+P, they still don’t have the country risk rating level on them. Also the formatting in the overview section is all over the place. Would it be possible to align the currency heading with the currency and close all the gaps. Also the line on the population kind of appears out of nowhere, maybe this would be better placed with the information below?
 
@Russell-Healix , do you have any thoughts on this?
 
Just to let you know, our first concur client will be going live next week.

![image001](https://cloud.githubusercontent.com/assets/2427859/5819239/3c9907e2-a0b4-11e4-88be-327455b6a0e4.jpg)

**Comment by antonyjsmith** [comment]
The population is just a piece of text on a new line and is part of the overview [example attached] so not much I can do about that. There are always going to limitations in what we can do around formatting of HTML in the email as it will vary greatly between devices so it's best to keep it as simple as possible.
![screen shot 2015-01-20 at 14 56 48](https://cloud.githubusercontent.com/assets/2427859/5819339/d572de66-a0b4-11e4-8900-db5a8cecf1e8.png)

**Comment by Russell-Healix** [comment]
I'm happy enough with the population bit but I think Sam has a point with the other items.

**Comment by antonyjsmith** [comment]
Yeah, I'll definitely look at the formatting again and make some changes
